package.json

```
@@ -8,8 +8,11 @@
     "build": "npm-run-all --parallel \"build:*\"",
     "build:backend": "esbuild ./src/backend/index.ts --bundle --platform=node --outfile=dist/backend/index.cjs",
     "build:frontend": "vite build",
-    "dev": "docker compose up --watch --build",
-    "dev:edusharing": "dotenvx run --env-file=.env -- tsx --watch src/edusharing-mock/start-server.ts",
+    "dev-prep": "npm-run-all --parallel --parallel \"dev-prep:*\"",
+    "dev-prep:frontend": "vite build --watch",
+    "dev-prep:databases": "docker compose up --build",
```

```suggestion
    "dev-prep:databases": "docker compose up --detach --build",
```
I never looked at the logs from mongodb or mysql in all this time. I think we can suppress them to remove some noise from the console.

---

NIce, thank you. See my comment below.